homeassistant/components/geniushub/climate.py

```diff
@@ -26,6 +31,31 @@
 
 GH_ZONES = ["radiator", "wet underfloor"]
 
+ATTR_ZONE_MODE = "mode"
+ATTR_DURATION = "duration"
+
+SVC_SET_ZONE_MODE = "set_zone_mode"
+SVC_SET_ZONE_OVERRIDE = "set_zone_override"
+
+SET_ZONE_MODE_SCHEMA = vol.Schema(
+    {
+        vol.Required(ATTR_ENTITY_ID): cv.entity_id,
```

Comment:
The entity_id item should not be included when registering the service with out entity service helper. It will be includedby the helper when it generates the entity service schema.